context/app/static/js/components/detailPage/provenance/ProvTableDerivedLink/style.js

```diff
@@ -6,8 +6,6 @@ const LinkButton = styled.a`
   background-color: ${(props) => props.theme.palette.primary.main};
   font-size: ${(props) => props.theme.typography.subtitle2.fontSize};
   font-weight: ${(props) => props.theme.typography.subtitle2.fontWeight};
-  margin-top: ${(props) => props.theme.spacing(1)}px;
-  width: 175px;
```

Comment:
What motivated this change? I'm somewhat worried about unintended consequences of removing a hard coded width.

Reply:
The button size was fixed width which was affecting the alignment with the text above it.